tests/test_json_schema.py

```diff
@@ -2484,7 +2484,7 @@ class Model(BaseModel):
     assert Model.model_json_schema() == {
         'properties': {
             'x': {
-                'items': {'type': 'integer'},
+                'maxItems': 2,
```

Comment:
@dmontagu did we add `maxItems` on purpose? This changed but I'm not sure why since json schema is a pedantic thing, not a core thing

Reply:
See line ~800 in `json_schema.py` - `items` should be present with `extras_schema`, but the code is probably still checking for `extra_schema`.

Reply:
Thought I got those, will check

Reply:
done